frontend/src/components/pages/CompleteProfilePage.tsx

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import Select from "react-select";
+import {
+  Box,
+  Button,
+  Flex,
+  FormControl,
+  Input,
+  Heading,
+  IconButton,
+  Text,
+  Textarea,
+} from "@chakra-ui/react";
+import { Icon } from "@chakra-ui/icon";
+import { MdFolder } from "react-icons/md";
+import Header from "../navigation/Header";
+
+const DropdownIndicator = () => {
+  return (
+    <Box
+      borderLeft="5px solid transparent"
+      borderRight="5px solid transparent"
+      borderTop="9px solid black"
+      margin="10px"
+    />
+  );
+};
+
+const CompleteProfilePage = () => {
+  const onSubmitClick = () => {
+    window.location.href = `/`;
+  };
+
+  return (
+    <Flex direction="column" height="100vh">
+      <Header />
+      <Flex direction="row" flex={1} marginLeft="50px">
+        <Flex direction="column" margin="40px" flex={1} maxWidth="50%">
+          <Heading size="lg">Complete your profile!</Heading>
+          <Text marginBottom="20px">
+            For the purposes of this platform, please enter your details
+          </Text>
+          <FormControl>
+            <Heading size="sm" marginTop="24px">
+              Full name*
+            </Heading>
+            <Input type="name" />
+            <Heading size="sm" marginTop="24px">
+              Email*
+            </Heading>
+            <Input type="email" />
+            <Heading size="sm" marginTop="24px">
+              Educational qualifications*
+            </Heading>
+            <Textarea type="qualifications" height="150px" />
+            <Heading size="sm" marginTop="24px">
+              Language experience*
+            </Heading>
+            <Textarea type="experience" height="150px" />
+          </FormControl>
+          <Heading size="sm" marginTop="24px">
+            Select language*
+          </Heading>
+          <Text marginBottom="12px">
+            Select the language that you would like to start with. Additional
+            languages can be added after signing up.
+          </Text>
+          <Box width="60%">
+            <Select
+              placeholder="Select language"
+              components={{ DropdownIndicator }}
+            />
+          </Box>
+        </Flex>
+        <Flex direction="column" marginTop="160px" flex={1} marginLeft="60px">
+          <Heading size="sm">Upload your CV (optional)</Heading>
+          <Flex
+            direction="column"
+            border="2px"
+            borderColor="rgba(29, 108, 165, 0.4)"
+            maxWidth="80%"
+            height="250px"
+            rounded="md"
+            bg="#e8f0f6"
```

Comment:
this is actually `blue.100` at 10% opacity
![image](https://user-images.githubusercontent.com/15113249/145422336-0469e75f-2b3d-4749-9d8f-0080e65fc4eb.png)
also nit: use "background" instead of "bg" - more readable